File: 0001 Code Chef/0024 Lucky Colver/implementation/0004 java/LuckyClover.java
import java.util.Scanner;

public class LuckyClover{
    public static void main(String[] args){
        Scanner read = new Scanner(System.in);
        String stringInput;
        stringInput = read.nextLine();
        int n = Integer.parseInt(stringInput);
        int totalLeaves = 4 + 3*(n-1);
        System.out.println(totalLeaves);
    }
}

File: 0001 Code Chef/0024 Lucky Colver/implementation/0005 kotlin/LuckyClover.kt
fun main(){
    var stringInput:String? = readLine()
    if(stringInput != null){
        var n:Int? = stringInput.toIntOrNull()
        if(n != null){
            var totalLeaves:Int = 4 + 3*(n-1);
            println(totalLeaves);
        }
    }
}